smcontroller/smcontroller.go

```diff
@@ -302,8 +276,10 @@ func (controller *Controller) RegisterSM(stream pb.SMService_RegisterSMServer) e
 	nodeCfg := launcher.NodeInfo{
 		NodeInfo: cloudprotocol.NodeInfo{
 			NodeID: nodeConfig.NodeConfiguration.NodeId, NodeType: nodeConfig.NodeConfiguration.NodeType,
-			NumCPUs: nodeConfig.NodeConfiguration.NumCpus, TotalRAM: nodeConfig.NodeConfiguration.TotalRam,
-			Partitions: make([]cloudprotocol.PartitionInfo, len(nodeConfig.NodeConfiguration.Partitions)),
+			SystemInfo: cloudprotocol.SystemInfo{
```

Comment:
not related message. Please remove chages, i will make the same while integration. Or move it to separate commit